Phoenix/PHSpace.m

```diff
@@ -61,8 +66,18 @@ @implementation PHSpace
 void CGSRemoveWindowsFromSpaces(CGSConnectionID connection, CFArrayRef windowIds, CFArrayRef spaceIds);
 
 // XXX: Undocumented private API to move the given windows (CGWindowIDs) to the given space
```

Comment:
```suggestion
// XXX: Undocumented private API to move the given windows (CGWindowIDs) to the given space, only works prior to macOS 14.5
```

Reply:
Done! `clang-format` demands it be broken up into two lines though,  so I added a line break after the comma.